app/controllers/staff/funds_controller.rb

```diff
@@ -43,12 +45,12 @@ def create
   end
 
   def edit
-    @fund = policy_scope(Fund).find(id)
+    @fund = Fund.find(id)
```

Comment:
I think this would make more sense if the style could be applied to all actions. As it is some actions will have it in one line whilst the others will have to have it on multi-line eg.

```
def create
    @fund = Fund.new(fund_params)
    @organisation = Organisation.find(organisation_id)
    @fund.organisation = @organisation

    authorize @fund
```

I'm in favour of the original which keeps it is slightly more verbose but keeps it consistent.